### DISCUSSION TITLE
Problem at changing date in file 3_cover.tex

### DISCUSSION THREAD

**Discussion by dansyndicate** [joaomlourenco/novathesis#465] [Q&A] [answered]
I am trying to change the date in the cover by changing the code in the file 3_cover.tex, but it is not working.
Please help me.

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/ef64422b-1bd8-45ad-b62d-4ba48f7f1b2a)
![image](https://github.com/joaomlourenco/novathesis/assets/129518837/20f5522e-f903-42a1-98ac-6eefb969b77a)

**Comment by joaomlourenco** [comment, marked as answer]
If this a working version?  A submission (provisional) version? Or the final version?
Please change `\ntsetup{docstatus=final}` in `Config/1_novathesis.tex` accordingly.

**Reply by dansyndicate** [reply]
Thank you professor, now it is working.
